docs/source/setup.rst

```diff
@@ -201,6 +201,91 @@ See the :doc:`EC2 quickstart <ec2>` for information on the ``dask-ec2`` easy
 setup script to launch a canned cluster on EC2.
 
 
+Cloud
+-----
+
+It is easy to launch a Dask cluster and Jupyter notebook server on cloud
+resources using Kubernetes_ and Helm_.
+
+.. _Kubernetes: https://kubernetes.io/
+.. _Helm: https://helm.sh/
+
+
+Setup Cluster
+~~~~~~~~~~~~~
+
+First you will need to set up an account with one of Google, Amazon, or
+Microsoft Azure,  and launch a cluster running Kubernetes.  This is
```

Comment:
Although it isn't "Cloud", I found minikube helpful for testing things out locally before trying on Google. Both `helm install stable/dask-distributed` and `helm install stable/spark` worked for me locally.

Reply:
Cool.  I wasn't aware of minikube.  Should we mention here in these docs or is this just an FYI?

Reply:
Not sure... I think a parenthetical "or minikube for local testing" may be worthwhile to ease the burden of entry for people looking to test things out. It seems to be the officially blessed version of "local kubernetes", so I don't see much downside.